src/node/index.d.ts

```diff
@@ -0,0 +1,526 @@
+declare module 'grpc' {
+  import { ReflectionObject, Service } from 'protobufjs'
+
+  /**
+   * Default options for loading proto files into gRPC
+   */
+  export interface ILoadOptions {
+    /**
+     * Load this file with field names in camel case instead of their original case. Defaults to `false`.
+     */
+    convertFieldsToCamelCase: boolean;
+
+    /**
+     * Deserialize bytes values as base64 strings instead of buffers. Defaults to `false`.
+     */
+    binaryAsBase64: boolean;
+
+    /**
+     * Deserialize long values as strings instead of objects. Defaults to `true`.
+     */
+    longsAsStrings: boolean;
+
+    /**
+     * Deserialize enum values as strings instead of numbers. Defaults to `true`.
+     */
+    enumsAsStrings: boolean;
+  }
+
+  export interface ILoadProtobufOptions extends ILoadOptions {
+    /**
+     * Indicate that an object from the corresponding version of
+     * ProtoBuf.js is provided in the value argument. If the option is 'detect',
+     * gRPC will guess what the version is based on the structure of the value.
+     * Defaults to 'detect'.
+     */
+    protobufjsVersion: 5 | 6 | 'detect';
+  }
+
+  /**
+   * Load a gRPC object from a `.proto` file.
+   * @param filename The file to load
+   * @param format The file format to expect. Defaults to 'proto'.
+   * @param options Options to apply to the loaded file.
+   * @return The resulting gRPC object.
+   */
+  export function load(filename: string, format?: 'proto' | 'json', options?: ILoadOptions): IProtobufDefinition;
+
+  /**
+   * Map from `.proto` file.
+   * - Namespaces become maps from the names of their direct members to those member objects
+   * - Service definitions become client constructors for clients for that service. They also have a service member that can be used for constructing servers.
+   * - Message definitions become Message constructors like those that ProtoBuf.js would create
+   * - Enum definitions become Enum objects like those that ProtoBuf.js would create
+   * - Anything else becomes the relevant reflection object that ProtoBuf.js would create
+   */
+  export interface IProtobufDefinition {
+    [key: string]: any;
```

Comment:
You can probably tighten this a bit. [This](https://github.com/mixer/etcd3/blob/password-auth/src/types/grpc.d.ts#L121) is what worked for me